quickstart.ipynb

```diff
@@ -461,12 +461,11 @@
    "metadata": {},
    "source": [
     "## OMF Request Responses\n",
-    "In the output from preceeding code you should see an operation-id. When posting OMF messages to AVEVA Data Hub, a successful request returns a 202 Accepted response code along with an operation-id. This response indicates that the supplied JSON was accepted as valid and that the request has been submitted for processing. This response is from the OMF API accepting the request body, the actual OMF message is still being processed by AVEVA Data Hub and a successful processing of the message is not guaranteed at this point. \n",
+    "In the output from preceeding code you should see an operation-id. This response is from the OMF API accepting the request body, the actual OMF message is still being processed by AVEVA Data Hub and a successful processing of the message is not guaranteed at this point. \n",
     "\n",
-    "If there are any issues with the data flow, contact AVEVA Technical Support with this operation-id. \n",
+    "To determine if an OMF message is truly valid, send the message to the CONNECT Data Services OMF Validation endpoint (../api/v1/omf/validation). A successful response indicates that the OMF message is valid.",
```

Comment:
Would it be better to hyperlink to the [Omf/Validation ](https://docs.aveva.com/bundle/connect-data-services-developer/page/api-reference/data-ingress/omf-ingress/omf-ingress-validation.html )endpoint docs?